python/mp/effects/__init__.py

```diff
@@ -23,5 +23,6 @@
     'looper',
     'wheel',
     'roulette',
-    'random_fill'
+    'random_fill',
+    'trigger_hijack_demo'
 ]
```

Comment:
If we had more time I'd ask "lordy is there no way to just import all the classes and be done with it?"